packages/components/Avatar/src/useAvatar.ts

```diff
@@ -135,10 +135,13 @@ export const removeRedundantCharacters = (name: string): string[] => {
   if (!name) {
     return [];
   }
+
+  const _name = name.length > 100 ? name.trim().substring(0, 100) : name;
```

Comment:
We can discuss how many symbols the name can contain

Reply:
100 seems like a reasonable limit.